app/javascript/gobierto_visualizations/webapp/containers/contracts/ContractsShow.vue

```diff
@@ -91,24 +125,60 @@ export default {
       process_type: '',
       permalink: '',
       assignee_routing_id: '',
-      labelAsignee: I18n.t('gobierto_visualizations.visualizations.contracts.assignee'),
-      labelTenderAmount: I18n.t('gobierto_visualizations.visualizations.contracts.tender_amount'),
-      labelContractAmount: I18n.t('gobierto_visualizations.visualizations.contracts.contract_amount'),
-      labelStatus: I18n.t('gobierto_visualizations.visualizations.contracts.status'),
-      labelProcessType: I18n.t('gobierto_visualizations.visualizations.contracts.process_type'),
-      labelContractType: I18n.t('gobierto_visualizations.visualizations.contracts.contract_type'),
-      labelPermalink: I18n.t('gobierto_visualizations.visualizations.contracts.permalink')
+      contractor: '',
+      contract_type: '',
+      start_date: '',
+      end_date: '',
+      award_date: '',
+      batch_number: '',
+      minor_contract: '',
+      open_proposals_date: '',
+      submission_date: '',
+      cpvs: '',
+      category_title: '',
+      labelAwardingEntity: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.awarding_entity') || '',
+      labelAssigneeDescription: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.assignee_description') || '',
+      labelContractAmount: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.contract_amount') || '',
+      labelTender: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.tender') || '',
+      labelAwarding: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.awarding') || '',
+      labelBidDescription: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.bid_description') || '',
+      labelBiddersDescription: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.bidders_description') || '',
+      labelStatus: I18n.t('gobierto_visualizations.visualizations.contracts.status') || '',
+      labelCategory: I18n.t('gobierto_visualizations.visualizations.subsidies.category') || '',
+      labelProcess: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.process') || '',
+      labelType: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.type') || '',
+      filterContractsBatches: []
+    }
+  },
+  computed: {
+    hasBatch() {
+      return this.batch_number > 0
+    },
+    isNotAMinorContract() {
+      return this.minor_contract === 'f'
+    },
```

Comment:
Don't you think it's less wordy; and more readable, to have a positive clause?
```suggestion
    isMinorContract() {
      return this.minor_contract === 't'
    },
```

Reply:
Dininini!